repository/z.json

```
@@ -158,6 +158,16 @@
 				}
 			]
 		},
+		{
+			"name": "Zig Unofficial",
```

Per your changes to the installation instructions and the `edit_settings` command, this should be:
```suggestion
			"name": "ZigUnofficial",
```